Fixing spinner load more

H-Lo opened the pull request:
Fixing issue:
https://github.com/vitmalina/w2ui/issues/2381

vitmalina commented on src/query.js:
```diff
@@ -519,6 +519,12 @@ class Query {
     show() {
         return this.toggle(true)
     }
+    showTr() {
```
query here is taken from another project, https://github.com/vitmalina/mquery. It should not be modified just here. It should also be modified in the original project. Also, I do not think adding `showTr` is a good idea. It should be still within `show` possibly check if element is a TR, then do display: 'table-row'

H-Lo replied:
Well yes, everything you said is right. But so far I don't know enough about mQuery to fix it. Tried to look into but not that I'm expert on JavaScript. Maybe I can take a loog again in few days and see is there a chance for me to fix it.

H-Lo commented:
@vitmalina
Ok, I think I did it right this time:
1) fixed `mquery.js` in `mquery` project to handle TR diferrently than other elements
2) reverted everything about `showTr()` in `w2ui` project
3) made changes to `query.js` inside `w2ui` project to be the same as the on in `mquery` project
